tests/test.config.ts

```diff
@@ -0,0 +1,14 @@
+export const environment = {
+  NODE_ENV: 'test',
+  SERVICEURL: 'https://apistore.be',
```

Comment:
Hier misschien eerder localhost of een *.local adres zetten, zodat er in de toekomst niet per ongeluk calls naar apistore.be gelanceerd gaan worden.